cincinnati/src/plugins/internal/edge_add_remove.rs

```diff
@@ -186,15 +193,17 @@ impl EdgeAddRemovePlugin {
         graph
             .find_by_metadata_key(&format!("{}.{}", self.key_prefix, "next.add"))
             .into_iter()
-            .try_for_each(|(from, from_string, to_csv)| -> Fallible<()> {
-                for to_string in to_csv.split(',').map(str::trim) {
-                    if let Some(to) = graph.find_by_version(&to_string) {
-                        info!("[{}]: adding {} {}", &from_string, "next", &to_string);
-                        handle_add_edge!("next", from, to, from_string, to_string);
+            .try_for_each(|(from, from_version, to_csv)| -> Fallible<()> {
+                for to_version in to_csv.split(',').map(str::trim) {
+                    let to_version = try_annotate_semver_build(&mut graph, to_version, &from)?;
```

Comment:
Same here - I'm not sure if this replacement is intended, but it would probably complicate debugging